README.md

```diff
@@ -45,168 +66,42 @@ dependencies {
-  override fun decode(value: CustomTypeValue<*>): Date {
-    return try {
-      DATE_FORMAT.parse(value.value.toString())
-    } catch (e: ParseException) {
-      throw RuntimeException(e)
-    }
-  }
-    
-  override fun encode(value: Date): CustomTypeValue<*> {
-    return GraphQLString(DATE_FORMAT.format(value))
-  }
-}
-    
-ApolloClient.builder()
-  .serverUrl(serverUrl)
-  .addCustomTypeAdapter(CustomType.DATE, dateCustomTypeAdapter)
-  .build()
-```
-
-## IntelliJ Plugin
-
-The [JS Graphql IntelliJ Plugin](https://jimkyndemeyer.github.io/js-graphql-intellij-plugin/) provides auto-completion, error highlighting, and go-to-definition functionality for your `.graphql` files. You can create a [`.graphqlconfig`](https://jimkyndemeyer.github.io/js-graphql-intellij-plugin/docs/developer-guide#working-with-graphql-endpoints-and-scratch-files) file to use GraphQL scratch files to work with your schema outside product code (such as to write temporary queries to test resolvers).
-
-## Releases
-
-Our [changelog](https://github.com/apollographql/apollo-android/releases) has the release history. 
-
-Releases are hosted on [jcenter](https://jcenter.bintray.com/com/apollographql/apollo/).
-
 Latest development changes are available in Sonatype's snapshots repository:
 
-```gradle
+```gradle:title=build.gradle.kts
 repositories {
-  maven { url 'https://oss.sonatype.org/content/repositories/snapshots/' }
+  maven { 
+    url = uri("https://oss.sonatype.org/content/repositories/snapshots/")
+  }
 }
 ```
 
-## Advanced topics
-
-Advanced topics are available in [the official docs](https://www.apollographql.com/docs/android/):
+## Requirements 
 
-* [caching.md](https://www.apollographql.com/docs/android/essentials/caching/)  
-* [plugin-configuration.md](https://www.apollographql.com/docs/android/essentials/plugin-configuration/) 
-* [android.md](https://www.apollographql.com/docs/android/advanced/android/) 
-* [file-upload.md](https://www.apollographql.com/docs/android/advanced/file-upload/)
-* [coroutines.md](https://www.apollographql.com/docs/android/advanced/coroutines/) 
-* [rxjava2.md](https://www.apollographql.com/docs/android/advanced/rxjava2/)
-* [rxjava3.md](https://www.apollographql.com/docs/android/advanced/rxjava3/)
-* [persisted-queries.md](https://www.apollographql.com/docs/android/advanced/persisted-queries/)
-* [no-runtime.md](https://www.apollographql.com/docs/android/advanced/no-runtime/) 
-* [migrations.md](https://www.apollographql.com/docs/android/essentials/migration/)
+Apollo Android runs on the following platforms:
 
+* Android API level 15+
+* JDK 8+
+* iOS 13
+ 
 ## Contributing
 
 If you'd like to contribute, please see [Contributing.md](https://github.com/apollographql/apollo-android/blob/master/Contributing.md).
 
+## Additional resources
+
+- [#125, Fragmented Podcast](http://fragmentedpodcast.com/episodes/125/): Why's and How's about Apollo Android and the entire journey.
```

Comment:
Would be nice to also put a link to our slides. 🤘

Reply:
💯 I just added them